src/backend/dx12/src/native.rs

```diff
@@ -447,13 +447,13 @@ impl HalDescriptorPool<Backend> for DescriptorPool {
             };
         }
 
-        DescriptorSet {
+        Ok(DescriptorSet {
```

Comment:
the OOM error would go from failing `alloc_handles`, although it's fine to do later on